resources/defaults.css

```
@@ -21,6 +21,7 @@ body {
   will-change: content;
 }
 
+:lang(ja) .text,
 .lang-ja .text {
```

```suggestion
:lang(ja) .text {
```

Sorry for any confusing comment.  I think internal to cactbot we should standardize on one format.  I think `lang(ja)` makes sense to me.  Can cactbot CSS just use `lang(ja)` style?

My earlier comment was me saying that we still needed to keep adding the `lang-ja` class, which you also did.  However, I don't think we need to reference it in cactbot itself.

---

Yeah, I think I misunderstand your thought, thanks!